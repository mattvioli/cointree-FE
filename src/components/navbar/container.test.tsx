import React from "react";
import { render, screen } from "@testing-library/react";
import { Container } from "./container";

describe("Container component", () => {
  it("renders children correctly", () => {
    const childText = "This is the content inside the container.";
    render(<Container>{childText}</Container>);

    const containerElement = screen.getByText(childText);
    expect(containerElement).toBeInTheDocument();
  });

  it("has the correct Tailwind classes applied", () => {
    const childText = "This is the content inside the container.";
    render(<Container>{childText}</Container>);

    const containerElement = screen.getByText(childText);

    expect(containerElement).toMatchSnapshot();
  });
});
