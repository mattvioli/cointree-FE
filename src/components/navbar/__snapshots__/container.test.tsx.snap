// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container component has the correct Tailwind classes applied 1`] = `
<div
  class="w-full max-w-6xl"
>
  This is the content inside the container.
</div>
`;
